docs/current_docs/api/terminal.mdx

```diff
@@ -0,0 +1,83 @@
+---
+slug: /api/terminal
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Interactive Terminal
+
+Dagger provides an interactive terminal that can help greatly when trying to debug a pipeline failure.
+
+To use this, set one or more explicit breakpoints in your Dagger pipeline with the `Container.terminal()` method. Dagger then starts an interactive terminal session at each breakpoint. This lets you inspect a `Directory` or a `Container` at any point in your pipeline run, with all the necessary context available to you.
```

Comment:
You can also introduce this with `dagger core`. For example:
```
dagger core container from --address=alpine terminal
```

Reply:
Added